R/utils.R

```diff
@@ -295,14 +295,14 @@ pruneViewportAdjustmentFromBins <- function(bins, xVP, x, viewport) {
   return(bins)
 }
 
-findBinStart <- function(x) {
+findBinStart <- function(x, addTimeZone = c(FALSE, TRUE)) {
   if (!length(x)) {
      return(character(0))
   }
 
   if (all(grepl(" - ",x))) {
     x <- veupathUtils::strSplit(x, " - ")
-    x <- paste0(x,'T00:00:00')
+    if (addTimeZone) x <- paste0(x,'T00:00:00')
```

Comment:
In the split below (and for `findBinEnd`)

the split pattern would be safer as `",\\s*"` - due to the recent post-processing of numeric bin range labels to include a space